README.md

```diff
@@ -18,6 +18,23 @@ SPIR-V is defined by the Khronos Group Inc.
 See the [SPIR-V Registry][spirv-registry] for the SPIR-V specification,
 headers, and XML registry.
 
+## Verisoning SPIRV-Tools
+
+See [`CHANGES`](CHANGES) for a high level summary of recent changes, by version.
+
+SPIRV-Tools project version numbers are of the form `v`*year*`.`*index* and with
```

Comment:
The `v` prefix is kinda redundant for me; it stands for `version` and should not be part of the version number. (We say `version 2016.0` or `v2016.0` instead of `version v2016.0` to mean version `2016.0`.)